`test_partial_evict_tenant` is flaky

See https://neon-github-public-dev.s3.amazonaws.com/reports/pr-3975/release/4631307967/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/86bde5a71b3be727/

```
AssertionError: the other tenant should be evicted to is min_resident_size, i.e., max layer file size
assert 10813440 < (2 * 5242880)
 +  where 5242880 = EvictionEnv(timelines=[(`TenantId("a7cdffb428510c7e804d76eb26cfa11e"), TimelineId("01d2589e75a4adeb9920ceda13fa26f7"))...a7cdffb428510c7e804d76eb26cfa11e"): Lsn("0/4AA79E0"), `TenantId("9952fa59f9ad4d531fdd720dc88aebd1"): Lsn("0/6464048")}).layer_size
```

Triggered in #3975

From @bayandin's gist:

```
test_runner/regress/test_disk_usage_eviction.py:404: in test_partial_evict_tenant
    assert (
E   AssertionError: the other tenant should be evicted to is min_resident_size, i.e., max layer file size
E   assert 10813440 < (2 * 5242880)
E    +  where 5242880 = EvictionEnv(timelines=[(`TenantId("c7e4c7ab15acb88542b4dab042b67ade"), TimelineId("2aab1324c55877a45f89bf2d900cf57e"))...c7e4c7ab15acb88542b4dab042b67ade"): Lsn("0/4AA3940"), `TenantId("2daf96411ba1ab91a410b538b899da1f"): Lsn("0/6464048")}).layer_size
```

- [test_partial_evict_tenant[debug-pg14]](https://neon-github-public-dev.s3.amazonaws.com/reports/main/4991705948/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/4952bb4fddc29fb8/)
- [test_partial_evict_tenant[release-pg14]](https://neon-github-public-dev.s3.amazonaws.com/reports/main/4983948954/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/6deaeec432f5a74c/)

Interesting:
- https://neon-github-public-dev.s3.amazonaws.com/reports/pr-5198/6311728531/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/cb5381d334697d75
- https://neon-github-public-dev.s3.amazonaws.com/reports/pr-5198/6311728531/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/9daeac88a7328995/retries

(debug and release or release and debug runs).

Messages:

```
AssertionError: the other tenant should be evicted to is min_resident_size, i.e., max layer file size
assert 12615680 < (2 * 5242880)
 +  where 5242880 = EvictionEnv(timelines=[(`TenantId("8f59ff83aa35cb932291d413d08888f6"), TimelineId("b6bf7a5373cef9450f436bc87203120a")), (`TenantId("fe5aa14efc939f5cda53601c4252bbee"), TimelineId("4ac9ba7dfa1440c6fd20ec66f4bd10e1"))], neon_env=<fixtures.neon_fixtures.NeonEnv object at 0x7f9d6f82e2e0>, pg_bin=<fixtures.neon_fixtures.PgBin object at 0x7f9d6f82e670>, pageserver_http=<fixtures.pageserver.http.PageserverHttpClient object at 0x7f9d6f8a1e80>, layer_size=5242880, pgbench_init_lsns={`TenantId("8f59ff83aa35cb932291d413d08888f6"): Lsn("0/7A515B8"), `TenantId("fe5aa14efc939f5cda53601c4252bbee"): Lsn("0/ACC2EC8")}).layer_size
```

```
AssertionError: the other tenant should be evicted to is min_resident_size, i.e., max layer file size
assert 12607488 < (2 * 5242880)
 +  where 5242880 = EvictionEnv(timelines=[(`TenantId("7938da0f418db29172d0006c46e85981"), TimelineId("d0482b82818338b7e4052a16f9674104")), (`TenantId("5652dd761d6674b6d3cf1d8b1b4a0539"), TimelineId("db8294cb8566853b9e9d6a87949247e3"))], neon_env=<fixtures.neon_fixtures.NeonEnv object at 0x7f4786f081c0>, pg_bin=<fixtures.neon_fixtures.PgBin object at 0x7f47870fcc10>, pageserver_http=<fixtures.pageserver.http.PageserverHttpClient object at 0x7f4786f08dc0>, layer_size=5242880, pgbench_init_lsns={`TenantId("7938da0f418db29172d0006c46e85981"): Lsn("0/7A515B8"), `TenantId("5652dd761d6674b6d3cf1d8b1b4a0539"): Lsn("0/ACC0E78")}).layer_size
```

Where:
- `2 * 5242880` would had been `10485760` or the familiar 10MiB
- 12607488 = 5259264 + 5259264 + 2088960

warmed up tenant `7938da0f418db29172d0006c46e85981/d0482b82818338b7e4052a16f9674104` went from 125MiB (132055040) to 70MiB (74260480).

other tenant `5652dd761d6674b6d3cf1d8b1b4a0539/db8294cb8566853b9e9d6a87949247e3` went from  177MiB (186023936) to 12MiB (12607488).

Report formatted:

```
2023-09-26T11:15:03.126956Z  INFO request{method=PUT path=/v1/disk_usage_eviction/run request_id=3345cb95-466e-4139-9470-2dedd659b55c}: disk_usage_eviction_task_iteration_impl finished res=Ok(Finished(IterationOutcomeFinished {
before: Usage { config: Config { evict_bytes: 231079936 }, freed_bytes: 0 },
planned: PlannedUsage {
  respecting_tenant_min_resident_size: Usage {
    config: Config { evict_bytes: 231079936 },
    freed_bytes: 231211008
  },
  fallback_to_global_lru: None
},
assumed: AssumedUsage { projected_after: Usage {
  config: Config { evict_bytes: 231079936 },
  freed_bytes: 231211008
},
failed: LayerCount { file_sizes: 0, count: 0 } } }))
```

I think this run completed successfully but for the passing runs the size of small layers on the smaller timeline is different:
- 655360
- 647168

Interesetingly enough, the larger tenant has exactly the same sized layers on both runs.

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -403,8 +403,8 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     du_by_timeline = env.du_by_timeline()
 
     # pick any tenant
-    [our_tenant, other_tenant] = list(du_by_timeline.keys())
-    (tenant_id, timeline_id) = our_tenant
+    [warm, cold] = list(du_by_timeline.keys())
+    (tenant_id, timeline_id) = warm
 
     # make our tenant more recently used than the other one
     env.warm_up_tenant(tenant_id)
@@ -412,9 +412,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     # Build up enough pressure to require evictions from both tenants,
     # but not enough to fall into global LRU.
     # So, set target to all occupied space, except 2*env.layer_size per tenant
-    target = (
-        du_by_timeline[other_tenant] + (du_by_timeline[our_tenant] // 2) - 2 * 2 * env.layer_size
-    )
+    target = du_by_timeline[cold] + (du_by_timeline[warm] // 2) - 2 * 2 * env.layer_size
     response = ps_http.disk_usage_eviction_run({"evict_bytes": target})
     log.info(f"{response}")
 
@@ -429,29 +427,29 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
             later_tenant_usage < du_by_timeline[tenant]
         ), "all tenants should have lost some layers"
 
-    warmed_size = later_du_by_timeline[our_tenant]
+    warm_size = later_du_by_timeline[warm]
 
-    # bounds for warmed_up size
-    warmed_lower = 0.5 * du_by_timeline[our_tenant]
+    # bounds for warmed_size
+    warm_lower = 0.5 * du_by_timeline[warm]
 
     # We don't know exactly whether the cold tenant needs 2 or just 1 env.layer_size wiggle room.
     # So, check for up to 3 here.
-    warmed_upper = warmed_lower + 3 * env.layer_size
+    warm_upper = warm_lower + 3 * env.layer_size
 
-    other_size = later_du_by_timeline[other_tenant]
-    other_upper = 2 * env.layer_size
+    cold_size = later_du_by_timeline[cold]
+    cold_upper = 2 * env.layer_size
 
     log.info(
-        f"expecting for our_tenant: {human_bytes(warmed_lower)} < {human_bytes(warmed_size)} < {human_bytes(warmed_upper)}"
+        f"expecting for warm tenant: {human_bytes(warm_lower)} < {human_bytes(warm_size)} < {human_bytes(warm_upper)}"
     )
-    log.info(f"expecting for other_tenant: {human_bytes(other_size)} < {human_bytes(other_upper)}")
+    log.info(f"expecting for cold tenant: {human_bytes(cold_size)} < {human_bytes(cold_upper)}")
 
-    assert warmed_size > warmed_lower, "our warmed up tenant should be at about half size (lower)"
-    assert warmed_size < warmed_upper, "our warmed up tenant should be at about half size (upper)"
+    assert warm_size > warm_lower, "our warmed up tenant should be at about half size (lower)"
+    assert warm_size < warm_upper, "our warmed up tenant should be at about half size (upper)"
 
     assert (
-        other_size < other_upper
-    ), "the other tenant should be evicted to its min_resident_size, i.e., max layer file size"
+        cold_size < cold_upper
+    ), "the cold tenant should be evicted to its min_resident_size, i.e., max layer file size"
 
 
 def poor_mans_du(

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -419,24 +419,28 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
             later_tenant_usage < du_by_timeline[tenant]
         ), "all tenants should have lost some layers"
 
-    warmed_up = later_du_by_timeline[our_tenant]
-    assert (
-        warmed_up > 0.5 * du_by_timeline[our_tenant]
-    ), f"our warmed up tenant should be at about half capacity but was {human_bytes(warmed_up)} expected more than {human_bytes(0.5 * du_by_timeline[our_tenant])}"
-    assert (
-        # We don't know exactly whether the cold tenant needs 2 or just 1 env.layer_size wiggle room.
-        # So, check for up to 3 here.
-        warmed_up
-        < 0.5 * du_by_timeline[our_tenant] + 3 * env.layer_size
-    ), "our warmed up tenant should be at about half capacity, part 2"
+    warmed_size = later_du_by_timeline[our_tenant]
+
+    # bounds for warmed_up size
+    warmed_lower = 0.5 * du_by_timeline[our_tenant]
+
+    # We don't know exactly whether the cold tenant needs 2 or just 1 env.layer_size wiggle room.
+    # So, check for up to 3 here.
+    warmed_upper = warmed_lower + 3 * env.layer_size
 
     other_size = later_du_by_timeline[other_tenant]
+    other_upper = 2 * env.layer_size
 
     log.info(
-        f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}"
+        f"expecting for our_tenant: {human_bytes(warmed_lower)} < {human_bytes(warmed_size)} < {human_bytes(warmed_upper)}"
     )
+    log.info(f"expecting for other_tenant: {human_bytes(other_size)} < {human_bytes(other_upper)}")
+
+    assert warmed_size > warmed_lower, "our warmed up tenant should be at about half size (lower)"
+    assert warmed_size < warmed_upper, "our warmed up tenant should be at about half size (upper)"
+
     assert (
-        other_size < 2 * env.layer_size
+        other_size < other_upper
     ), "the other tenant should be evicted to its min_resident_size, i.e., max layer file size"
 
 

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -89,7 +89,7 @@ class EvictionEnv:
         """
         lsn = self.pgbench_init_lsns[tenant_id]
         with self.neon_env.endpoints.create_start("main", tenant_id=tenant_id, lsn=lsn) as endpoint:
-            self.pg_bin.run(["pgbench", "-S", endpoint.connstr()])
+            self.pg_bin.run(["pgbench", "--select-only", "--no-vacuum", endpoint.connstr()])
 
     def pageserver_start_with_disk_usage_eviction(
         self, period, max_usage_pct, min_avail_bytes, mock_behavior

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -74,11 +74,13 @@ class EvictionEnv:
     pgbench_init_lsns: Dict[TenantId, Lsn]
 
     def timelines_du(self) -> Tuple[int, int, int]:
-        return poor_mans_du(self.neon_env, [(tid, tlid) for tid, tlid in self.timelines])
+        return poor_mans_du(
+            self.neon_env, [(tid, tlid) for tid, tlid in self.timelines], verbose=False
+        )
 
     def du_by_timeline(self) -> Dict[Tuple[TenantId, TimelineId], int]:
         return {
-            (tid, tlid): poor_mans_du(self.neon_env, [(tid, tlid)])[0]
+            (tid, tlid): poor_mans_du(self.neon_env, [(tid, tlid)], verbose=True)[0]
             for tid, tlid in self.timelines
         }
 
@@ -445,7 +447,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
 
 
 def poor_mans_du(
-    env: NeonEnv, timelines: list[Tuple[TenantId, TimelineId]]
+    env: NeonEnv, timelines: list[Tuple[TenantId, TimelineId]], verbose: bool = False
 ) -> Tuple[int, int, int]:
     """
     Disk usage, largest, smallest layer for layer files over the given (tenant, timeline) tuples;
@@ -468,9 +470,11 @@ def poor_mans_du(
                 smallest_layer = min(smallest_layer, size)
             else:
                 smallest_layer = size
-            log.info(f"{tenant_id}/{timeline_id} => {file.name} {size}")
+            if verbose:
+                log.info(f"{tenant_id}/{timeline_id} => {file.name} {size}")
 
-        log.info(f"{tenant_id}/{timeline_id}: sum {total}")
+        if verbose:
+            log.info(f"{tenant_id}/{timeline_id}: sum {total}")
         total_on_disk += total
 
     assert smallest_layer is not None or total_on_disk == 0 and largest_layer == 0

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -402,8 +402,8 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
         < 0.5 * du_by_timeline[our_tenant] + 3 * env.layer_size
     ), "our warmed up tenant should be at about half capacity, part 2"
     assert (
-        later_du_by_timeline[other_tenant] < 2 * env.layer_size
-    ), "the other tenant should be evicted to is min_resident_size, i.e., max layer file size"
+        later_du_by_timeline[other_tenant] < 3 * env.layer_size
+    ), "the other tenant should be evicted to is 2-3 layers, i.e., max layer file size"
 
 
 def poor_mans_du(

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -471,10 +471,10 @@ def poor_mans_du(
             else:
                 smallest_layer = size
             if verbose:
-                log.info(f"{tenant_id}/{timeline_id} => {file.name} {size}")
+                log.info(f"{tenant_id}/{timeline_id} => {file.name} {size} ({human_bytes(size)})")
 
         if verbose:
-            log.info(f"{tenant_id}/{timeline_id}: sum {total}")
+            log.info(f"{tenant_id}/{timeline_id}: sum {total} ({human_bytes(total)})")
         total_on_disk += total
 
     assert smallest_layer is not None or total_on_disk == 0 and largest_layer == 0

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -388,7 +388,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
 
     # Build up enough pressure to require evictions from both tenants,
     # but not enough to fall into global LRU.
-    # So, set target to all occipied space, except 2*env.layer_size per tenant
+    # So, set target to all occupied space, except 2*env.layer_size per tenant
     target = (
         du_by_timeline[other_tenant] + (du_by_timeline[our_tenant] // 2) - 2 * 2 * env.layer_size
     )

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -89,7 +89,21 @@ class EvictionEnv:
         """
         lsn = self.pgbench_init_lsns[tenant_id]
         with self.neon_env.endpoints.create_start("main", tenant_id=tenant_id, lsn=lsn) as endpoint:
-            self.pg_bin.run(["pgbench", "--select-only", "--no-vacuum", endpoint.connstr()])
+            # instead of using pgbench --select-only which does point selects,
+            # run full table scans for all tables
+            with endpoint.connect() as conn:
+                cur = conn.cursor()
+
+                tables_cols = {
+                    "pgbench_accounts": "abalance",
+                    "pgbench_tellers": "tbalance",
+                    "pgbench_branches": "bbalance",
+                    "pgbench_history": "delta",
+                }
+
+                for table, column in tables_cols.items():
+                    cur.execute(f"select avg({column}) from {table}")
+                    _avg = cur.fetchone()
 
     def pageserver_start_with_disk_usage_eviction(
         self, period, max_usage_pct, min_avail_bytes, mock_behavior

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -432,7 +432,9 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
 
     other_size = later_du_by_timeline[other_tenant]
 
-    log.info(f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}")
+    log.info(
+        f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}"
+    )
     assert (
         other_size < 3 * env.layer_size
     ), "the other tenant should be evicted to 2-3 layers"

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -406,7 +406,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     [warm, cold] = list(du_by_timeline.keys())
     (tenant_id, timeline_id) = warm
 
-    # make our tenant more recently used than the other one
+    # make picked tenant more recently used than the other one
     env.warm_up_tenant(tenant_id)
 
     # Build up enough pressure to require evictions from both tenants,
@@ -444,8 +444,8 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     )
     log.info(f"expecting for cold tenant: {human_bytes(cold_size)} < {human_bytes(cold_upper)}")
 
-    assert warm_size > warm_lower, "our warmed up tenant should be at about half size (lower)"
-    assert warm_size < warm_upper, "our warmed up tenant should be at about half size (upper)"
+    assert warm_size > warm_lower, "warmed up tenant should be at about half size (lower)"
+    assert warm_size < warm_upper, "warmed up tenant should be at about half size (upper)"
 
     assert (
         cold_size < cold_upper

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -153,6 +153,7 @@ def human_bytes(amt: float) -> str:
 
     raise RuntimeError("unreachable")
 
+
 @pytest.fixture
 def eviction_env(request, neon_env_builder: NeonEnvBuilder, pg_bin: PgBin) -> EvictionEnv:
     """

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -403,7 +403,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     ), "our warmed up tenant should be at about half capacity, part 2"
     assert (
         later_du_by_timeline[other_tenant] < 3 * env.layer_size
-    ), "the other tenant should be evicted to is 2-3 layers"
+    ), "the other tenant should be evicted to 2-3 layers"
 
 
 def poor_mans_du(

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -436,8 +436,8 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
         f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}"
     )
     assert (
-        other_size < 3 * env.layer_size
-    ), "the other tenant should be evicted to 2-3 layers"
+        other_size < 2 * env.layer_size
+    ), "the other tenant should be evicted to its min_resident_size, i.e., max layer file size"
 
 
 def poor_mans_du(

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -141,6 +141,18 @@ class EvictionEnv:
         self.neon_env.pageserver.allowed_errors.append(".*WARN.* disk usage still high.*")
 
 
+def human_bytes(amt: float) -> str:
+    suffixes = ["", "Ki", "Mi", "Gi"]
+
+    last = suffixes[-1]
+
+    for name in suffixes:
+        if amt < 1024 or name == last:
+            return f"{int(round(amt))} {name}B"
+        amt = amt / 1024
+
+    raise RuntimeError("unreachable")
+
 @pytest.fixture
 def eviction_env(request, neon_env_builder: NeonEnvBuilder, pg_bin: PgBin) -> EvictionEnv:
     """
@@ -406,17 +418,22 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
             later_tenant_usage < du_by_timeline[tenant]
         ), "all tenants should have lost some layers"
 
+    warmed_up = later_du_by_timeline[our_tenant]
     assert (
-        later_du_by_timeline[our_tenant] > 0.5 * du_by_timeline[our_tenant]
-    ), "our warmed up tenant should be at about half capacity, part 1"
+        warmed_up > 0.5 * du_by_timeline[our_tenant]
+    ), f"our warmed up tenant should be at about half capacity but was {human_bytes(warmed_up)} expected more than {human_bytes(0.5 * du_by_timeline[our_tenant])}"
     assert (
         # We don't know exactly whether the cold tenant needs 2 or just 1 env.layer_size wiggle room.
         # So, check for up to 3 here.
-        later_du_by_timeline[our_tenant]
+        warmed_up
         < 0.5 * du_by_timeline[our_tenant] + 3 * env.layer_size
     ), "our warmed up tenant should be at about half capacity, part 2"
+
+    other_size = later_du_by_timeline[other_tenant]
+
+    log.info(f"later_du_by_timeline[other_tenant] = {other_size} ({human_bytes(other_size)}), before_relax = {other_size < 2 * env.layer_size}")
     assert (
-        later_du_by_timeline[other_tenant] < 3 * env.layer_size
+        other_size < 3 * env.layer_size
     ), "the other tenant should be evicted to 2-3 layers"
 
 

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -403,7 +403,7 @@ def test_partial_evict_tenant(eviction_env: EvictionEnv):
     ), "our warmed up tenant should be at about half capacity, part 2"
     assert (
         later_du_by_timeline[other_tenant] < 3 * env.layer_size
-    ), "the other tenant should be evicted to is 2-3 layers, i.e., max layer file size"
+    ), "the other tenant should be evicted to is 2-3 layers"
 
 
 def poor_mans_du(

--- a/test_runner/regress/test_disk_usage_eviction.py
+++ b/test_runner/regress/test_disk_usage_eviction.py
@@ -244,8 +244,12 @@ def test_broken_tenants_are_skipped(eviction_env: EvictionEnv):
 
     healthy_tenant_id, healthy_timeline_id = env.timelines[1]
 
-    broken_size_pre, _, _ = poor_mans_du(env.neon_env, [(broken_tenant_id, broken_timeline_id)])
-    healthy_size_pre, _, _ = poor_mans_du(env.neon_env, [(healthy_tenant_id, healthy_timeline_id)])
+    broken_size_pre, _, _ = poor_mans_du(
+        env.neon_env, [(broken_tenant_id, broken_timeline_id)], verbose=True
+    )
+    healthy_size_pre, _, _ = poor_mans_du(
+        env.neon_env, [(healthy_tenant_id, healthy_timeline_id)], verbose=True
+    )
 
     # try to evict everything, then validate that broken tenant wasn't touched
     target = broken_size_pre + healthy_size_pre
@@ -253,8 +257,12 @@ def test_broken_tenants_are_skipped(eviction_env: EvictionEnv):
     response = env.pageserver_http.disk_usage_eviction_run({"evict_bytes": target})
     log.info(f"{response}")
 
-    broken_size_post, _, _ = poor_mans_du(env.neon_env, [(broken_tenant_id, broken_timeline_id)])
-    healthy_size_post, _, _ = poor_mans_du(env.neon_env, [(healthy_tenant_id, healthy_timeline_id)])
+    broken_size_post, _, _ = poor_mans_du(
+        env.neon_env, [(broken_tenant_id, broken_timeline_id)], verbose=True
+    )
+    healthy_size_post, _, _ = poor_mans_du(
+        env.neon_env, [(healthy_tenant_id, healthy_timeline_id)], verbose=True
+    )
 
     assert broken_size_pre == broken_size_post, "broken tenant should not be touched"
     assert healthy_size_post < healthy_size_pre

